.test-infra/jenkins/job_PostRelease_NightlySnapshot.groovy

```diff
@@ -23,9 +23,6 @@ import CommonJobProperties as commonJobProperties
 job('beam_PostRelease_NightlySnapshot') {
   description('Runs post release verification of the nightly snapshot.')
 
-  // Execute concurrent builds if necessary.
-  concurrentBuild()
```

Comment:
Why are you removing it now? And why did we add it before? (I think it makes sense to remove, just curious what made you do it).

Reply:
The tests do operations on gcp resources such as bigquery tables and verify results there. Stop concurrent building will prevent multiple tests working on the same table that cause conflicts and failures.

Reply:
Ah that makes sense. Thanks.